selfdrive/car/docs_definitions.py

```diff
@@ -129,9 +129,9 @@ def get_column(self, column: Column, star_icon: str, footnote_tag: str) -> str:
     elif column == Column.MODEL and len(self.years):
       item += f" {self.years}"
 
-    footnote = get_footnote(self.footnotes, column)
-    if footnote is not None:
-      item += footnote_tag.format(self.all_footnotes[footnote])
+    footnotes = get_footnotes(self.footnotes, column)
+    if footnotes is not None:
+      item += ", ".join(footnote_tag.format(self.all_footnotes[fn]) for fn in footnotes)
```

Comment:
It would look strange to delimit superscript text with a normal comma.  I recommend `𝄒` character as delimeter.

Reply:
or else use Wikipedia style with brackets

![Screen Shot 2022-07-22 at 11 26 09 PM](https://user-images.githubusercontent.com/1708631/180570848-5aa805ac-2960-4f2d-a808-3c0ad5b8313b.png)

Reply:
We don't link to specific footnotes, so I think commas are fine if they match the font style of the superscript (which they do now)